pwa/src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,6 @@
+@import "../../styles/setting";
+
+footer {
+  background-color: $burgundy;
+  color: #ffffff;
```

Comment:
Please extract this color code as a variable in `colors.scss` under common `styles` directory to increase reusability.